### ISSUE TITLE
Question of loss and training

### ISSUE THREAD

**Issue by FUNGERN** [csdwren/PReNet#15]
Hello, thank you for your contribution. I have read your thesis, can you answer some questions?
1. Does Loss only use SSIM and not MSE? What happens if these two Loss are used at the same time?
2. About four networks. During training, are these 4 written separately when typing commands?
For example to train PReNet, I just need to type 

> bash train_PReNet.sh

and I don't need to type the remaining three

> bash train_PRN.sh
> bash train_PReNet_r.sh
> bash train_PRN_r.sh

Looking forward to your answer, thank you

**Comment by csdwren** [comment]
1. We do not try the combinaton of loss functions. Maybe hybrid loss functions with careful trade-off parameter tuning will lead to better results. 
2. yes.

**Comment by FUNGERN** [comment]
@csdwren Thank you for your reply, I'm reading your network code, and two more questions,
1. Is the convolution an ordinary convolution?
The parameters in 2.nn.Conv2d (6, 32, 3, 1, 1) represent
in_channels = 6 out_channels = 32 kernel_size = 3 stride = 1 padding = 1

**Comment by csdwren** [comment]
yes

**Comment by FUNGERN** [comment]
thanks
